HerPortal.UnitTests/HerPortal.UnitTests.csproj

```diff
@@ -8,6 +8,7 @@
 
     <ItemGroup>
         <PackageReference Include="FluentAssertions" Version="6.6.0" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="6.0.16" />
```

Comment:
(Where did this come from?)

Reply:
I added it to resolve a build warning in the tests